locale/en_US/translations.ts

```diff
@@ -734,5 +734,53 @@
             <translation>Set the maximum amount of days a show should stay in the 'Next Up' list without watching it.</translation>
             <extracomment>Settings Menu - Description for option</extracomment>
         </message>
+        <message>
+            <source>Playback Information</source>
+            <translation>Playback Information</translation>
+        </message>
+        <message>
+            <source>Transcoding information</source>
+            <translation>Transcoding information</translation>
```

Comment:
Keeping capitalization consistent with the rest
```suggestion
            <source>Transcoding Information</source>
            <translation>Transcoding Information</translation>
```